packages/reactotron-app/App/Shared/CommandToolbar.js

```diff
@@ -153,6 +158,22 @@ class CommandToolbar extends Component {
       const text = apiToMarkdown(this.props.command.payload)
       clipboard.writeText(text)
     } catch (e) {
+      clipboard.writeText(body)
```

Comment:
`body` isn't defined in this function, so we'd end up throwing a new exception.

Reply:
probably best to leave this empty too...

Reply:
Doh, this was a failed merge conflict resolution. Will fix. Thanks for catching.